payments/models.py

```diff
@@ -0,0 +1,6 @@
+from django.db import models
+
+
+class ProcessedTransaction(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
```

Comment:
Since this app is already depended on konfera why dont we use KonferaModel, as it already have date_created field...?